django_graph_api/graphql/types.py

```diff
@@ -52,9 +52,10 @@ class Field(object):
     creation_counter = 0
     arguments = {}
 
-    def __init__(self, description=None, arguments=None):
+    def __init__(self, description=None, arguments=None, null=True):
         self.arguments = arguments or {}
         self.description = description
+        self.nullable = null
```

Comment:
I think that it might be easier for folks to get used to the API if the attribute were also called `null`